cd/application-deployment/dev/vaec-api-task-definition.json

```diff
@@ -231,20 +231,20 @@
           "name": "COMP_AND_PEN_MESSAGES_ENABLED",
           "value": "True"
         },
-        {
-          "name": "COMP_AND_PEN_SERVICE_ID",
-          "value": "d6aa2c68-a2d9-4437-ab19-3ae8eb202553"
-        },
-        {
-          "name": "COMP_AND_PEN_TEMPLATE_ID",
-          "value": "fca415e8-ff5f-4c29-a121-6dd534aa90c7"
-        },
         {
           "name": "COMP_AND_PEN_DYNAMODB_NAME",
           "value": "dev-bip-msg-mpi-profile-participant-cache-table"
         }
       ],
       "secrets": [
+        {
+          "name": "COMP_AND_PEN_SERVICE_ID",
+          "value": "arn:aws-us-gov:ssm:us-gov-west-1:171875617347:parameter/dev/notification-api/comp-and-pen/service-id"
```

Comment:
Should we just remove these things from the API task definitions if they are only necessary in celery task definitions?

Reply:
We had a previous convo about that, so I was surprised to see them all still being used in every task definition. What was the outcome of that previous convo?

Reply:
Got it, we have another ticket to audit task defs. I'm fine with this going out as it stands then.